Design and technology subject page (#4140)
DFE-Digital/get-into-teaching-app: app/views/content/life-as-a-teacher/explore-subjects/design-and-technology.md

---
title: Become a design and technology teacher
title_paragraph: |-
  <p>

>   As a design and technology teacher, you can empower pupils to turn their ideas into reality. You can encourage pupils to understand how things work, why they matter, and how they could be improved.</p>

Reviewer: ```suggestion
As a design and technology teacher, you'll empower pupils to turn their ideas into reality. You'll help pupils to understand how things work, why they matter, and how they could be improved.</p>
```
